matthannigan.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Matt Hannigan
+---
+# Hi
+I'm Matt Hannigan from Pittsburgh.
+
+## What I do
+I work at [The Sprout Fund](http://www.sproutfund.org).
+
+## My favorite things
+- grantmaking
+- decisionmaking
+- spreadsheets
+- legos
```

Comment:
I'm not very good at the [SNOT technique](http://www.brickwiki.info/wiki/SNOT_techniques).